pkg/api/v1/resource/helpers.go

```diff
@@ -24,8 +24,16 @@ import (
 
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+// ContainerType signifies container type. This is replicated here to avoid an import cycle.
+type ContainerType int
```

Comment:
I wonder if we should just declare it in types.go so it can be used universally without worrying of dependencies.